Read the VCS version from a local file instead of using git.
We create this file in the `post_patch()` step in the srcpkg
template.

diff -u -r a/src/meson.build b/src/meson.build
--- a/src/meson.build	2024-12-16 06:11:30.000000000 +0000
+++ b/src/meson.build	2024-12-22 10:29:49.654509579 +0000
@@ -171,8 +171,8 @@
 compiler_name = cc.get_id()
 compiler_version = cc.version()
 
-vcs_tag_cmd = ['git', 'describe', '--always', '--tags', '--dirty=+']
+vcs_tag_cmd = ['cat', '../voidlinux_pkg_version.txt']
 vcs_tag = run_command(vcs_tag_cmd, check: false).stdout().strip()
 version_tag = vcs_tag + ' (' + compiler_name + ' ' + compiler_version + ')'
 
 gamescope_version_conf = configuration_data()

